### ISSUE TITLE
Leveraging a 'contributes to' annotation for an ISA?

### ISSUE THREAD

**Issue by robnash** [geneontology/go-annotation#3242]
The budding yeast gene SPO11 (P23179; https://www.yeastgenome.org/locus/S000001014), currently has a MF annotation to: GO:0016889 endodeoxyribonuclease activity, producing 3’-phosphomonoesters. This annotation was recently disputed by another curator as they said it sounds more like a DNA topoisomerase type II (double strand cut, ATP-hydrolyzing). After reviewing the literature it appears there is no direct demonstration of a topo II activity for the yeast gene. SPO11 by sequence appears to be a member of a type II DNA topoisomerase family present in archaea, termed DNA topoisomerase VI (TopoVI), which also contains fission yeast rec12 and worm spo-11. These enzymes are considered to be distinct from the classical type II DNA topoisomerases, so were classified as an evolutionary distinct type IIB DNA topoisomerase family. An activity was purified from _Sulfolobus shibatae_ consisting of a heterotetramer (PMID:7961685), and was later reconstituted in _E. coli_ (PMID: 9801313). It consists of the SPO11 archaebacterial homolog, top6a (https://www.uniprot.org/uniprot/O05208), and top6b (https://www.uniprot.org/uniprot/O05207). Since both subunits are required for ATP-dependent relaxation, decatenation and double-strand cleavage of DNA a 'contributes to' annotation to MF term GO:0003918 DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) activity would be required, and both subunits would need to be annotated to the CC term: GO:0009330 DNA topoisomerase type II (double strand cut, ATP-hydrolyzing) complex.

If these annotations were made, the question is could we at SGD leverage this annotation to make an ISA annotation? I can find alignments in several papers (PMID: 11809802, fig 1C; and PMID: 9121560) between _Sulfolobus_ top6A and yeast SPO11. However, since the _Sulfolobus shibatae_ activity requires both top6A and top6B for this activity can we still leverage the MF 'contributes to' annotation for an ISA since its unknown whether yeast SPO11 acts by itself or requires a second subunit. I am not aware of a 'contributes to' using ISA. Otherwise we would need to remove the current annotation and replace with GO:0045027 DNA end binding, which is much less satisfying

Any thoughts? 

Cheers,
Rob

**Comment by pgaudet** [comment]
I think so - in PAINT we carry over any relevant qualifier

@vanaukenk @ValWood does that seem OK to you ? We should document this. 

Thanks, Pascale

**Comment by ValWood** [comment]
Actually, in this example, I wouldn't use "contributes_to" because we *know* it isn't the catalytic subunit.

I only use "contributes_to" if the active site is "distributed"  (for example polymerases and the proteasome)

I wouldn't use it for a  cyclin, for example with "protein kinase activity". Here I would use  a "kinase regulator" term.

I always think this is safer,  because some software strips the qualifiers. My rule of thumb is that the annotation should still look reasonable if the qualifier is removed. 

Here, I would use "endonuclease regulator" but I would still want a bit more evidence, I think.

"Since both subunits are required for ATP-dependent relaxation, decatenation and double-strand cleavage of DNA " does not really imply a contribute_to  the endonuclease activity (it could just be directly upstream rather than a regulatory subunit).

In summary, I don't think we really know the function here. I'm tempted to just have "DNA end binding" for now...

**Comment by ValWood** [comment]
I have also mailed a couple of people who work on rec12 for input.

**Comment by ValWood** [comment]
Lovely reply from Gerry Smith:

Hi Val,

You’re right:  Rec12 and Spo11 are not endonucleases.  Rather, they are phosphotransferases, much like Topo I, to which they are related (Topo VI is closer, however).  Rec12 becomes covalently linked to DNA, and that requires Tyr 98 (our work).  The same is true for Spo11 of S. cerevisiae, which I think was shown to make the covalent link via a Tyr and a phospho group.  I think either phosphotransferase or DSB-forming protein (but not enzyme, as it is non-catalytic) is appropriate.  Please let me know if this is not clear or if you wish more info.

Thanks again for this and all the other wonderful things you do for us pombe people in the trenches (what it feels like these days).

Best wishes,

Gerry

 (could not resist leaving in the last line, the community  feedback makes our job worthwhile...)
I will try to find more about which publications are appropriate to capture this detail.

**Comment by ValWood** [comment]
More info from Gerry, for S. cerevisiae annotations.

You asked for citations for Rec12 or Spo11 being covalently linked via Tyr to DNA.  I wrote to Scott Keeney, who first published S. cerevisiae Spo11 being linked to DNA, and he sent refs.  As far as he and I know, the direct chemistry has not been done as had been done for TopoI and TopoII, but Keeney et al. Cell 1997 cites the linkage being alkali-resistant (Tyr, but not Ser or Thr, linkages are) and Matt Neale shows it is sensitive to TDP2, an enzyme specific for Tyr-phosphodiesters (https://www.biorxiv.org/content/10.1101/2019.12.18.881268v1).  I think you should cite these two papers plus Cromie et al., PLoS Genet 2007 and Fowler et al. Genome Res 2014.

 So in summary  it seems we should avoid any endonuclease annotation, they seem like a bit of a 'red herring', and just go with the currrently  available info from yeast publications.  I removed the  endonuclease ISS 's for PomBase.

**Comment by suzialeksander** [comment]
assigning @robnash so hopefully he gets notifications

**Comment by robnash** [comment]
Thanks for the helpful comments and for forwarding the info. from the rec12 experts! With respect to subunits in yeast, I cannot find any information on additional subunits or a complex. In the archaea case I am pretty convinced that this is a 'contributes to' situation because purified renatured A subunit requires the purified B subunit for the ATP-dependent relaxation, decatenation and double-strand cleavage of DNA, in a reconstituted system, so probably not an upstream effect. I emailed UniProt with the references so they can potentially add the experimental annotations for the Sulfolobus shibatae genes in case these are of use later.

For the yeast SPO11 gene, I am going to go the conservative route, removing the 'endodeoxyribonuclease activity, producing 3'-phosphomonoesters' (GO:0016889) annotation and adding 'DNA end binding' (GO:0045027) using the Kenney paper references (PMID: 9039264). I will also update the gene description.
